examples/src/main/python/sql/arrow.py

```diff
@@ -275,6 +275,28 @@ def merge_ordered(left: pd.DataFrame, right: pd.DataFrame) -> pd.DataFrame:
     # +--------+---+---+----+
 
 
+def arrow_python_udf_example(spark: SparkSession) -> None:
+    from pyspark.sql.functions import udf
+
+    @udf(returnType='int')  # A default, pickled Python UDF
+    def slen(s):  # type: ignore[no-untyped-def]
+        return len(s)
+
+    @udf(returnType='int', useArrow=True)  # An Arrow Python UDF
+    def add_one(x):  # type: ignore[no-untyped-def]
+        if x is not None:
+            return x + 1
```

Comment:
Why don't we use same function as an example for both case to avoid confusion? Or maybe it's intended to use different example for some reason??

Reply:
Good idea! Used same udf body.